_content/how_we_work.yml

```diff
@@ -1,39 +1,52 @@
 page_title:
   heading_title: "How we work"
   heading_text: |-
-     USDS deploys small, responsive groups of technology experts to work with and empower civil servants. These multi-disciplinary teams bring best practices and new approaches to untangle some of our nation’s most important problems.
+     USDS deploys small, responsive groups of technologists to work with and empower our nation’s leading civil servants. Our designers, engineers, product managers, bureaucracy hackers, and acquisition strategists bring best practices and new approaches to support government modernization.
 
   heading_button_text: "Learn about our mission"
 
 cops:
-  heading_title: "Recruiting the country’s best technologists for ‘tours of service’"
+  heading_title: "Building a team with a broad range of experiences to transform government."
   heading_text: |-
-     Our staff comes from all corners of the technology industry, nonprofit world, and government to serve ‘tours' of service, bringing a steady influx of fresh perspectives into government. Tours typically last between six months and two years, with a maximum length of four years.
+     With tours of service lasting no more than four years, the U.S. Digital Service brings fresh perspectives on technology and delivery to government. To build the best possible team, we focus on the same values in our hiring as we do in our work.
```

Comment:
question about
> U.S. Digital Service brings fresh perspectives on technology and delivery to government. 

I'm curious why "to government" would be appropriate here, but "to the government" used in this example, in the Design CoP description:
> We often introduce human-centered design practices to the government.